apps/test/unit/templates/rubrics/RubricContainerTest.jsx

```diff
@@ -218,9 +245,7 @@ describe('RubricContainer', () => {
         />
       </Provider>
     );
-    await act(async () => {
-      await Promise.resolve();
-    });
+    await wait();
```

Comment:
I got tired of copy and pasting this... I do wonder what in the world of RTL could help, but I just used this pattern there, too. It still seems useful when you want to delay the action when it happens initially or based on the passage of time.